### ISSUE TITLE
Code highlighting breaks when using return with text content

### ISSUE THREAD

**Issue by LsHallo** [meanmail-dev/nginx-intellij-plugin#7]
**Describe the bug**
Code highlighting breaks when using return with text content.

**To Reproduce**
Paste this config:
```conf
server {
    listen 80 default_server;
    listen [::]:80 default_server;

    location / {
        add_header Content-Type "text/plain";
        return 200 "LB";
    }

    location ~ ^(.*)\.rss {
        resolver 127.0.0.11 ipv6=off;
        rewrite ^ $request_uri break;
        return 400;
        proxy_pass http://nginx_target$uri;
    }
}
```
Code highlighting breaks after the return 200 "LB" block:
![image](https://user-images.githubusercontent.com/22171518/140977150-e7e5e790-fde9-4898-8d99-eef95424fdb0.png)

**Expected behavior**
Code highlighting will continue to work further down in the config file.

**IDE (please complete the following information. See Main Menu -> Help -> About):**
 - OS: Windows 10 Pro 64bit
 - Product: WebStorm
 - Version: 2021.2.3
 - JRE Version: OpenJDK 64-Bit Server VM by JetBrains s.r.o
 - Project SDK: -

**Plugin (please complete the following information. See Main Menu -> File -> Settings -> Plugins):**
 - Version: 2021.2-212

**Additional Information**
`return code [text]` is supported:
http://nginx.org/en/docs/http/ngx_http_rewrite_module.html#return

**Comment by michaelboyles** [comment]
Same problem with `proxy_set_header Connection "upgrade";`

Don't know if the quotes are strictly necessary, but that's the exact form used by the docs http://nginx.org/en/docs/http/websocket.html

**Comment by meanmail** [comment]
Thank you for reporting this issue! The fix has been implemented and will be available in version 2024.5, which will be released soon. The update includes enhanced syntax highlighting for strings, braces, values and other elements.
